app/tasks/mail.js

```diff
@@ -0,0 +1,34 @@
+const nodemailer = require('nodemailer');
+const createWelcomeMail = require('../email_templates/welcome.js');
+const dotenv = require('dotenv');
+dotenv.config();
+
+module.exports = async (payload, helpers) => {
```

Comment:
same thing with `payload`, if we can destructure or type that would make this more robust